test/image-prepare

```diff
@@ -188,10 +188,6 @@ def main():
     if args.quick:
         customize.append("--quick")
 
-    # rpms/debs can be built offline; arch can't yet
-    if args.image != 'arch':
-        customize.append("--no-network")
```

Comment:
Inverted logic -- you need to *always* specify `--no-network` now. This switches back all other OSes to online.

I was puzzled how the [arch test here](https://logs.cockpit-project.org/logs/pull-17151-20220318-154439-ecffe463-arch/log.html) was able to build.

Reply:
That was silly, do we potentially want to allow an online build via a flag in image-customize?

Reply:
The default already is "online build". `--no-network` disables that.

Reply:
You dropped the conditional `--no-network`, but you need  to add it unconditionally somewhere. Preferably right to its initial value.

Reply:
I've added it directly to `customize` now, is that ok?

Reply:
Yes, looks fine. Thanks!